tests/e2e/e2e_test.go

```diff
@@ -73,6 +73,29 @@ func pingSvc(url string) {
 	}
 }
 
+func pingCmd(cmd string) bool {
+	pingTimeout := time.After(5 * time.Minute)
+	tick := time.Tick(time.Second)
+
+	for {
+		select {
+		case <-pingTimeout:
+			Fail("timeout out after 5 minutes")
```

Comment:
Shouldn't be this timeout shorter? 5mins is really long time.
How about 1min? Is there a chance that after 1min the command will succeed?

Reply:
Sure.  Have changed it to 1min for now. ALso, have changed the function name to `waitForCmdOut`